### DISCUSSION TITLE
vant中 index.less 是如何被打包成 index.css 的

### DISCUSSION THREAD

**Discussion by lixiaofa** [youzan/vant#11553] [Q&A]
![image](https://user-images.githubusercontent.com/23230215/217494756-1e08619d-2e65-488e-a32f-19683a1f5119.png)

**Comment by daiwanxing** [comment]
`less` 暴露了一个 [render](https://lesscss.org/usage/#programmatic-usage) 函数，可以将 less 文件编译成 css 文件。 在 Vant-cli 中，你可以找到 `compiler` 这个package，里面有个 `compiler-less` 的文件，这里就是负责将 vant 内的 less 代码编译成 css。
